Header header
string seed
string node_pow
string node_routing
